### PR TITLE
Fix system properties name

### PR DESCRIPTION
**Pull request by Vovchyk**
## Description

Fix the properties name used by code

Originated from https://github.com/rsksmart/rskj/pull/1535

## Motivation and Context

Fix issue #1533

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Comment by Vovchyk** [comment]
pipeline: run

**Comment by Vovchyk** [comment]
pipeline: run

File: rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java [hunk 1]
```diff
@@ -151,15 +151,15 @@ public double minerMinFeesNotifyInDollars() {
     }
 
     public boolean bloomServiceEnabled() {
-        return getBoolean("bloom.service", false);
+        return getBoolean("blooms.service", false);
```
Comment (marcciosilva) [inline rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java:154]: isn't there a test we could make to ensure the property is loaded correctly? like loading some fake system property and then getting it from these methods (maybe it's not worth it - but we didn't notice this wasn't working in the first place)

Comment (Vovchyk) [reply rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java:154]: good point. Done.

**Comment by Vovchyk** [comment]
pipeline: run

File: rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java [hunk 2]
```diff
@@ -34,21 +42,59 @@
     public void defaultValues() {
         Assert.assertFalse(config.isMinerClientEnabled());
```
Comment (marcciosilva) [inline rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java:43]: I'd use static imports for this as well (so it's the same as the changes you made)

Comment (Vovchyk) [reply rskj-core/src/test/java/co/rsk/config/RskSystemPropertiesTest.java]: done

**Review by marcciosilva — DISMISSED** [review]
lgtm!

**Review by aeidelman — APPROVED** [review]

**Pull request merged**